const sayError = () => {
    alert('Some error occurred!');
};

sayError();